org.nextcloud.Nextcloud.yml

```diff
@@ -6,7 +6,7 @@ runtime-version: '5.12'
 sdk: org.kde.Sdk
 
 command: nextcloud
-rename-desktop-file: nextcloud.desktop
+rename-desktop-file: com.nextcloud.desktopclient.nextcloud.desktop
```

Comment:
app id was changed in https://github.com/nextcloud/desktop/pull/1449
but not the name of the icon